src/pipelines/floodscan_pipeline.py

```diff
@@ -0,0 +1,349 @@
+import os
+
+import shutil
+from datetime import datetime
+from fileinput import filename
+from zipfile import ZipFile
+
+import pandas as pd
+import requests
+import xarray as xr
+
+from ..utils.azure_utils import download_from_azure, blob_client
+from ..utils.date_utils import create_date_range, get_datetime_from_filename, DATE_FORMAT
+from ..utils.raster_utils import invert_lat_lon
+
+from .pipeline import Pipeline
+
+SFED = "SFED"
```

Comment:
Could these be attributes of the `FloodScanPipeline` class? Eg. `self.sfed` and `self.mfed`? Regardless, might need a sentence or two on what these represent.